app/models/package.rb

```diff
@@ -111,6 +111,10 @@ def add_location(location_id)
     end
   end
 
+  def update_allow_web_publish
```

Comment:
actually the attribute name is allow_web_publish and i was following rails convention. I had thought of such ambiguity, anyway i'll rename it to make more sense.